benchmarks/bench_simul.c

```diff
@@ -157,6 +157,13 @@ static struct param_t {
 	{ 0 },
 };
 
+static struct {
+	os_cond_t cond;
+	os_mutex_t mutex;
+	uint64_t wanted;
+} ready;
+
+#ifndef VMEMCACHE_NO_STATS
```

Comment:
negation of negation...

Please reverse the logic.

Reply:
Done

Reply:
Nope, you just renamed it and I think the new name is even worse (REMOVE_STATS sounds like vmemcache will actively remove stats now and that's not true).

STATS_ENABLED?

Reply:
Done